x-pack/plugins/enterprise_search/public/applications/enterprise_search/routes.ts

```diff
@@ -5,5 +5,10 @@
  * 2.0.
  */
 
+import { docLinks } from '../shared/doc_links';
+
+export const DOCS_PREFIX = docLinks.enterpriseSearchBase;
+export const USERS_DOCS_URL = `${DOCS_PREFIX}/users-access.html`;
```

Comment:
This isn't 100% what this file should be used for. The `enterprise_search/` application is specifically for the Home/Overview plugin, and since this isn't being used there, I'd probably opt to put this in `shared/` instead - or just set it directly in `users_empty_prompt.tsx` instead of exporting it.

Reply:
https://github.com/elastic/kibana/pull/102826/commits/3bbd1aca55e033bebd883d7b5627148545dafdb6